client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -12,16 +14,12 @@ import Button from 'components/button';
 import Card from 'components/card';
 import Gridicon from 'components/gridicon';
 import analytics from 'lib/analytics';
```

Comment:
We could take advantage of the opportunity and also [move `analytics` to use Redux](https://github.com/Automattic/wp-calypso/blob/master/client/state/analytics/actions.js#L53).